cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -66,8 +66,10 @@
 import static io.ballerina.cli.utils.TestUtils.generateTesterinaReports;
 import static io.ballerina.cli.utils.TestUtils.loadModuleStatusFromFile;
 import static io.ballerina.cli.utils.TestUtils.writeToTestSuiteJson;
+import static org.ballerinalang.test.runtime.util.TesterinaConstants.DOT;
```

Comment:
Shall we use a more meaningful name such as `NAME_SEPARATOR` ?
Let's update the already existing constant :)